config/routes.rb

```diff
@@ -75,22 +75,24 @@
         end
       end
 
-      resources :criteria, except: [:index] do
+      resources :criteria do
         collection do
           post 'update_positions'
+          post 'upload_yml'
+          get  'download_yml'
         end
       end
 
-      resources :rubric_criteria, only: [:index] do
```

Comment:
I can get rid of this route and the one for flexible_criteria because they are not used at the moment. However, If I deleted them, I would have to delete the tests for the routes, which will have to be created back again if we decide to use csv later on. Should I delete them or comment them out?